README.md

```diff
@@ -1 +1,138 @@
-# proxy
\ No newline at end of file
+# Open Runtimes Proxy 🚦
+
+![open-runtimes-box-bg-cover](https://user-images.githubusercontent.com/1297371/151676246-0e18f694-dfd7-4bab-b64b-f590fec76ef1.png)
+
+---
+
+[![Discord](https://img.shields.io/discord/937092945713172480?label=discord&style=flat-square)](https://discord.gg/mkZcevnxuf)
+[![Build Status](https://github.com/utopia-php/balancing/actions/workflows/tester.yml/badge.svg)](https://github.com/utopia-php/balancing/actions/workflows/tester.yml)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+
+<!-- [![Docker Pulls](https://img.shields.io/docker/pulls/appwrite/appwrite?color=f02e65&style=flat-square)](https://hub.docker.com/r/appwrite/appwrite) -->
+
+Proxy server for [Open Runtimes](https://github.com/open-runtimes/open-runtimes), a runtime environments for serverless cloud computing for multiple coding languages.
+
+The proxy is responsible for checking health of executors, and proxying requests between them based on selected strategy, such as round-robin. Proxy is stateless and can be scaled horizontally when a load balanced is introduced in front of it.
+
+## Features
+
+* **Flexibility** - Adapter-driven approach allows switching between proxying strategies. You can even implement your own.
+* **Performance** - Coroutine-style HTTP servers allows asynchronous operations without blocking. We. Run. Fast! ⚡
+* **Open Source** - Released under the MIT license, free to use and extend.
+
+## Getting Started - Open Runtimes Executor
+
+> 🚧 Getting started with Open Runtimes Executor coming soon.
```

Comment:
Why do we need this section at all ? Shouldn't this be in the readme for the executor ?

Reply:
Good point, you would never know that proxy exists this way. We should have this in the executor readme.

Reply:
removing it from here.